src/site/content/en/blog/compat2021/index.md

```diff
@@ -118,7 +118,7 @@ have had issues with `auto-height` flex containers leading to incorrectly sized
 </div>
 
 
-[Igalia's Flexbox Cats](https://blogs.igalia.com/svillar/2021/01/20/flexbox-cats-a-k-a-fixing-images-in-flexbox/)
+[Igalia's flexbox Cats](https://blogs.igalia.com/svillar/2021/01/20/flexbox-cats-a-k-a-fixing-images-in-flexbox/)
```

Comment:
The title of this post was "Flexbox Cats", I don't think we should change this instance.